csu

```diff
@@ -173,14 +173,19 @@ function style {
   docker-compose run --rm -w /cs-unplugged/ django /docker_venv/bin/flake8
   pep8_status=$?
   printf "\nRunning Python docstring checker...\n"
-  docker-compose run --rm -w /cs-unplugged/ django /docker_venv/bin/pydocstyle --count --explain
+  docker-compose run --rm -w /cs-unplugged/ django /docker_venv/bin/pydocstyle --count --explain --match-dir='[^\.].*(?<!node_modules)'
```

Comment:
This is also raising a lot more `pydocstyle` errors as it's now checking migration files. Any exclusions should be set in `setup.cfg`.